lookout/style/format/quality_report_noisy.py

```diff
@@ -18,6 +21,11 @@
 from lookout.style.format.utils import prepare_files
 from lookout.style.format.virtual_node import VirtualNode
 
+
+REPOSITORIES = """https://github.com/warenlg/axios
+https://github.com/warenlg/jquery"""
```

Comment:
we should use a separate file for the data. But for now I think it is fine to leave a `TODO`

Reply:
This is how you do it here https://github.com/src-d/style-analyzer/pull/326/files#diff-517c2da73bf28591d05434b9d1b57f8bR28

Reply:
And a TODO: https://github.com/src-d/style-analyzer/pull/326/files#diff-517c2da73bf28591d05434b9d1b57f8bR26 :)

Reply:
and It was Egor.